docs/snapcraft-syntax.md

```diff
@@ -0,0 +1,84 @@
+# Syntax of the snapcraft.yaml file
+
+The `snapcraft.yaml` file is the main entry point to create a snap through
+Snapcraft. The main building blocks are parts and each defined part is
+independent from each other. In addition to parts, there are attributes
+that define the metadata for the snap package.
+
+What follows is a list of all the attributes the `snapcraft.yaml` file can
+contain.
+
+ * `name` (string)  
+   The name of the resulting snap.
+ * `version` (string)  
+   The version of the resulting snap.
+ * `vendor` (string)  
+   Vendor in email format as defined in RFC 5322.
```

Comment:
I think at some stage we'll just phase out the 1.x docs(?). With snapcraft 0.6 it's still required, right?